packages/core/admin/admin/src/components/AuthenticatedApp/index.js

```diff
@@ -110,8 +110,10 @@ const AuthenticatedApp = () => {
       userDisplayName={userDisplayName}
     >
       <RBACProvider permissions={permissions} refetchPermissions={refetch}>
-        <NpsSurvey />
-        <PluginsInitializer />
+        <>
+          <NpsSurvey />
+          <PluginsInitializer />
+        </>
```

Comment:
Why did you not just fix the prop-types? Imo there is no reason to use a wrapping fragment.

Reply:
That makes my bad didn't think about.